content/applications/marketing/email_marketing/analyze_metrics.rst

```diff
@@ -17,84 +20,182 @@ locations.
+  :menuselection:`Add Custom Group -->`:guilabel:`Responsible`, in the :guilabel:`Group By` column,
+  sorts all results *first* by the sent period, *then* by the team member responsible. This is a
+  useful metric for analyzing who on the team is sending in volume or quantity over a set time
+  period.
+
+  This can be verified by looking at the direction, and order, of the selections in the group tile
+  that appears in the search bar after the selections are made.
+
+.. example::
+   A monthly newsletter has been sent out, and 6.9% of the sent emails were bounced.
+
+   .. image:: analyze_metrics/newsletter-metrics.png
+      :align: center
+      :alt: The metrics smart buttons of the newsletter.
+
+   To see what these bounced recipients have in common, the records are grouped using a custom group
+   targeting :guilabel:`Mailing Lists`, which groups all records by the mailing lists they are on.
+   The records are then filtered using a custom filter with the rule `Created on >= 07/01/2024
+   00:00:00`, when the mailing list was last checked. This filter only includes recipients
+   that have been created on, or after, July 1st, 2024, in the report.
+
+      .. image:: analyze_metrics/metrics-filter.png
+         :align: center
+         :alt: The custom filter creation form.
+
+   Using these configurations, it is evident that all the recipients with bounced emails were added
+   after the list was last checked. Looking closer at the domains, it is evident that each
+   recipient has a malformed email domain (i.e: @yaoo.com instead of @yahoo.com), likely due to a
+   manual entry error while updating the database.
+
+   .. image:: analyze_metrics/malformed-addresses.png
+      :align: center
+      :alt: A list of bounced email addresses with malformed email domains.
+
+.. seealso::
+  View :doc:`../../essentials/search` for more information about making custom groups and
+  filters.
+
 .. _email-marketing/deliverability-issues:
 
+Mass mailing analysis
+=====================
+
+It is also possible to analyze the success between mailing campaigns by creating a *Mass Mailing
+Analysis* report. To begin, navigate to :menuselection:`Email Marketing app --> Reporting --> Mass
+Mailing Analysis`.
+
+A dashboard appears displaying a bar chart containing each mailing campaign. By default, *Sent* is
+selected, displaying the number of sent records on the y-axis. To change the measure, click the
+:guilabel:`Measures` button, and select the desired measure from the drop-down menu.
+
+.. example::
+   The following chart displays the number of opened emails from two different mass mailings.
+
+   In this view, it can be seen that the first mass mailing led to a higher opening rate than the
+   second. Because a lower opening rate can sometimes be attributed to a subject line that failed to
+   capture readers' attention, the subject line of each mass mailing can be a good place to begin
+   looking.
```

Comment:
Maintain consistent terminology :)
```suggestion
   In this view, it can be seen that the first mass mailing led to a higher opened rate than the
   second. Because a lower opened rate can sometimes be attributed to a subject line that failed to
   capture readers' attention, the subject line of each mass mailing can be a good place to begin
   looking.
```